CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+0.5.0 (2018-07-05)
+------------------
+
+- Switched to JSON API.
+- Set correct ticket status when creating a ticket
+- Added ability to modify default ticket type and status when calling
+  teamsupport.init()
+
```

Comment:
This version bump might also be a good time to drop support for python 3.3.

Reply:
3.3 is also specified in travis.yml https://github.com/yola/teamsupport-python/blob/f6dfe79e78b85a60ce43fbe5203dcd6f5b10f418/.travis.yml#L10-L11
and setup.py https://github.com/yola/teamsupport-python/blob/f6dfe79e78b85a60ce43fbe5203dcd6f5b10f418/setup.py#L59

Python 3.5 should probably also be added as a classifier in setup.py

Reply:
Added